### PR TITLE
Remove trailing white-space in files

### PR DESCRIPTION
**Pull request by ann0see**
**Short description of changes**
* Removes unneeded whitespace in all files (found by Atom text editor)
* Fixes line endings #2360 They should all be in unix format now

**Context: Fixes an issue?**
No, but hopefully stops cluttered diffs.
Fixes: #2360 
**Does this change need documentation? What needs to be documented and how?**
CHANGELOG: Internal: Remove unneeded white-space in all files and streamline line endings

**Status of this Pull Request**
Ready to be merged if CI is ok.

**What is missing until this pull request can be merged?**

## Checklist

- [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [ ] I tested my code and it does what I want
- [ ] My code follows the [style guide](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#source-code-consistency) 
- [ ] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
- [ ] I've filled all the content above

### CODE REVIEW

**Review by ann0see — COMMENTED** [review]
I think this is not dangerous to merge - maybe clang-format should handle this (probably it already does but not for the files here.)

**Review by pljones — APPROVED** [review]
I'd rather not include this in 3.8.2 now as it's not a blocker on releasing rc1, unlike the translations.

**Review by hoffie — APPROVED** [review]
Another potential whitespace fix to include: #2360.

**Comment by ann0see** [comment]
Could be done with dos2unix, I think?

**Comment by ann0see** [comment]
Ok @hoffie Included the fix as commit in this PR.

File: src/vstmain.cpp [hunk 1]
```diff
@@ -1,112 +1,112 @@
...
-    // connect timer event
-    connect ( &TimerOnOff, SIGNAL ( timeout() ), this, SLOT ( OnTimerOnOff() ) );
-
-    // clang-format off
-// TODO settings
-Client.SetServerAddr ( DEFAULT_SERVER_ADDRESS );
-    // clang-format on
-}
-
-bool CLlconVST::GetName ( char* cName )
```
Comment (ann0see) [inline src/vstmain.cpp:61]: llcon --> Jamulus

**Review by ann0see — COMMENTED** [review]
How much sense does it make to include unfinished files in main? Shouldn't we put them in a feature branch?

File: src/vstmain.cpp (hunk 1 above)
Comment (pljones) [reply src/vstmain.cpp:61]: The vst code _could_ also all get moved to a (dead) feature branch.

**Comment by ann0see** [comment]
Should not be squash merged.

File: src/vstmain.cpp (hunk 1 above)
Comment (ann0see) [reply src/vstmain.cpp:61]: Yes. I think that's best.

**Comment by ann0see** [comment]
Should be merged after upcoming release (not squash merged).

**Pull request merged**